command/rm.go

```diff
@@ -142,6 +150,22 @@ func newURLs(sources ...string) ([]*url.URL, error) {
 	return urls, nil
 }
 
+// srcURLsBucketVal checks whether given urls all correspond to the same
+// bucket or not.
+func srcURLsBucketVal(urls []*url.URL) error {
```

Comment:
`hasSameBuckets` is more readable. Also better return bool, and do error stuff at the callsite.